SpatialGDK/Source/SpatialGDK/Private/Interop/SkeletonEntityManifestPublisher.cpp

```diff
@@ -49,23 +49,27 @@ FManifestCreationHandle FSkeletonManifestPublisher::CreateManifestForPartition(I
 	{
 		AuthorityDelegation Authority;
 		Authority.Delegations.Emplace(SpatialConstants::SKELETON_ENTITY_MANIFEST_AUTH_COMPONENT_SET_ID, Partition);
+		Authority.Delegations.Emplace(SpatialConstants::SKELETON_ENTITY_MANIFEST_GSM_AUTH_COMPONENT_SET_ID,
+									  SpatialConstants::INITIAL_SNAPSHOT_PARTITION_ENTITY_ID);
 
 		SkeletonEntityManifestComponents.Emplace(
 			ComponentData(OwningComponentDataPtr(Authority.CreateComponentData().schema_type), AuthorityDelegation::ComponentId));
 	}
 
 	{
+		// Give GSM worker interest in this entity
+		SkeletonEntityManifestComponents.Emplace(ComponentData(SpatialConstants::SKELETON_ENTITY_MANIFEST_GSM_TAG_COMPONENT_ID));
```

Comment:
How does this give interest? There aren't component constraints here right?

Reply:
No - we're using self-interest here. GSM worker has auth over SKELETON_ENTITY_MANIFEST_GSM_TAG_COMPONENT_ID and then interest in SKELETON_ENTITY_MANIFEST_COMPONENT_ID